README.md

```diff
@@ -209,7 +209,7 @@ token = UserAuth('https://my.cerberus.url', 'username', 'password').get_token()'
 Generally it does NOT make sense to store Lambda secrets in Cerberus for two reasons:
 
 1. Cerberus cannot support the scale that lambdas may need, e.g. thousands of requests per second
-1. Lambdas will not want the extra latency needed to authenticate and read from Cerberus
+2. Lambdas will not want the extra latency needed to authenticate and read from Cerberus
```

Comment:
This is a README syntax that converts to the actually numbers for you. That's why you do all ones

Reply:
Updated. Thought I saw 2 1s in preview.